internal/backend/db/dbgorm/projects.go

```diff
@@ -48,6 +48,31 @@ func (gdb *gormdb) deleteProject(ctx context.Context, projectID sdktypes.UUID) e
 	})
 }
 
+func (gdb *gormdb) deleteProjectVars(ctx context.Context, id sdktypes.UUID) error {
+	// NOTE: should be transactional
+	db := gdb.db.WithContext(ctx)
+
+	// enforce foreign keys constrains while soft-deleting - should be no active deployments
+	var count int64
+	db.Model(&scheme.Deployment{}).Where("deleted_at is NULL and project_id = ?", id).Count(&count)
```

Comment:
looks like all deployments

Reply:
any non-deleted one that is

Reply:
```
// enforce foreign keys constrains while soft-deleting - should be no active deployments
```

the comment state it should check active deployments 

so which is wrong? code or comment ? 
I guess the comment since the context is delete env